script/Arcueid/Day 5/05_00_ARC05_2_2.txt

```diff
@@ -197,15 +197,15 @@ My resolve strengthens as a result.
 [sha:e9f08a3949642f8396d24b0558a474fa1abf7cbf]{
 -- Page 215, Offset 9129. Glued.
 -- それがあのホテルにいて自分だけ生き残った、俺が取るべき責任なんだと思う」
- I believe this is the responsibility I need to take as the only survivor from that hotel."
+I believe this is my responsibility as the sole survivor of that tragedy at the hotel."
```

Comment:
Dropped leading space?

Reply:
think i accidentally got rid of it :aichanteehee: